hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestDFSStripedInputStream.java

```diff
@@ -664,4 +666,70 @@ public void testUnbuffer() throws Exception {
       assertNull(in.parityBuf);
       in.close();
   }
+
+  @Test
+  public void testBlockReader() throws Exception {
+    ecPolicy = StripedFileTestUtil.getDefaultECPolicy(); // RS-6-3-1024k
```

Comment:
@jojochuang Thanks for your review. Since `TestDFSStripedInputStreamWithRandomECPolicy` extends `TestDFSStripedInputStream`, `testBlockReader` will fail on random EC policy in `TestDFSStripedInputStreamWithRandomECPolicy`. One way is to choose a fixed EC policy as here we do, another way is to extend test case to adapt any EC policy which can be considered too.

Reply:
ok. Thanks you Make sense.